src/hpr_routing.erl

```diff
@@ -2,7 +2,16 @@
 
 -export([
     init/0,
-    handle_packet/1
+    handle_packet/1,
+    routing_info_type/1
+]).
+
+-type routing_info() ::
```

Comment:
Maybe overkill, but these could also be records.

Reply:
Per @michaeldjeffrey , I may not have to pass this 'routing_info', so this question may become moot.

Reply:
This type is used in a couple of places. I don't see any advantage of making this into records.